### ISSUE TITLE
Request gang notifications and warnings upon creation

### ISSUE THREAD

**Issue by tnox** [DexterHuang/CyberCodeOnline#2344]
Not sure if this is the spot if not please direct me to the right place. 
Problem: Lost my gang
I purchased a gang for 50k and even contributed 10k more to when I saw the warning on the gang page that it was critical. I no longer have the gang and wanted to ask if we could put warnings on the gang page when buying a gang and in the tutorials about gangs to let people know what is needed to maintain them. Also would like a notification to be sent that the guild is in danger of being destroyed. If it was there I didn't see it and that is on me if its not there I would like to have it created. I don't know if I can get my money back but want to help new players so they don't lose their money for not knowing about anything I missed upon creating my gang. Thank you for your time.
TLDR: I paid for a gang but lost it not sure why. I would like to add to gang creation page and create a tutorial page to help new people. I don't know what I didn't do or I would make the page myself.

**Comment by tnox** [comment]
I created a pull request #2455 to fix this issue.
